pkg/rest/apisurface.go

```diff
@@ -368,6 +368,9 @@ func unpackUnbindRequest(r *http.Request) (*osb.UnbindRequest, error) {
 	vars := mux.Vars(r)
```

Comment:
Could you please add a test for this as well, similar to the https://github.com/pmorie/osb-broker-lib/blob/master/pkg/rest/apisurface_test.go#L11. Thanks! :)